### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by cainthebest** [gamedig/rust-gamedig#10] [General]
I would like to propose the creation of a roadmap for the repository, which would provide a clear and detailed plan for the development and maintenance of the codebase.

**Comment by CosminPerRam** [comment]
Hello, great suggestion!
Although I work on rust-gamedig in my spare time, I do think that having a roadmap could be good.
I will look into it today.

**Comment by cainthebest** [comment]
Sounds good, I originally wrote a lib for this that was closed source and worked on it since before the new year so i done quite a bit of research on the fundamentals of the lib. Id be happy to help out when i can.

**Comment by cainthebest** [comment]
Ive started some ideas to help with the roadmap, ill be adding more when i got the time.

**Reply by CosminPerRam** [reply]
I have made a roadmap, i'm waiting for @mmorrisontx to make it public, would be glad to add more from suggestions.

**Reply by CosminPerRam** [reply]
It's public!
